streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -461,6 +477,51 @@ public static void addInvocationRateAndCount(final Sensor sensor,
                                   "The average per-second number of " + operation);
     }
 
+    public static void addAmountRateAndTotalMetricsToSensor(final Sensor sensor,
```

Comment:
This is not part of the PR but I realized that `CumulativeCount` is in `streams.processor.internals.metrics` but not in `common.metrics.stats`. Is this intentional? @vvcephei

Reply:
IIRC, it was done to avoid a KIP.

Reply:
Ok, I took a closer look and had a bit of a flash-back to how confusing these metrics are... Maybe https://github.com/apache/kafka/pull/7057 will help.